### PR TITLE
javascript-conditionals - Tim Silva

### PR DESCRIPTION
**Pull request by silva-tim**
Screen Shot 2023-05-10 at 1 33 32 PM

### CODE REVIEW

File: javascript-conditionals/main.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+);
+console.log(
+  'isOldEnoughToDrinkAndDrive of 13 year old:',
+  isOldEnoughToDrinkAndDrive(person2)
+);
+
+function categorizeAcidity(pH) {
+  if (pH < 7 && pH >= 0) {
+    return 'acid';
+  } else if (pH > 7 && pH < 14) {
```
Comment (shawnkost) [inline javascript-conditionals/main.js:65]: 14 would still be considered a base, so this should be less than or equal to

**Review by shawnkost — CHANGES_REQUESTED** [review]
One small change here Tim!

File: javascript-conditionals/main.js (hunk 1 above)
Comment (silva-tim) [reply javascript-conditionals/main.js]: Screen Shot 2023-05-10 at 1 54 05 PM


Fixed

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**